Code/Legacy/CrySystem/SystemInit.cpp

```diff
@@ -1155,6 +1155,15 @@ bool CSystem::Init(const SSystemInitParams& startupParams)
     // Execute any deferred commands that uses the CVar commands that were just registered
     AZ::Interface<AZ::IConsole>::Get()->ExecuteDeferredConsoleCommands();
 
+#if defined (CARBONATED)
+    // Load the deffered level only after CrySystem has been fully initialized
+    AZ::TickBus::QueueFunction(
+       [this]()
+       { // Wait while RPI System is fully initialized
```

Comment:
It is not clear how RPI system initialization is connected with CrySystem init, can you please clarify in the comment?

Reply:
The comment has been changed to be more understandable